app.js

```diff
@@ -35,8 +35,11 @@ webserver.get("/healthz", async (req, res) => {
   // Check Database Connection
   try {
     const promisePool = pool.grabConnection();
-    await promisePool.ping();
-    status_checks.database = "OK";
+    if (promisePool) {
+      status_checks.database = "OK";
+    } else {
+      throw "pool didn't return a promise!";
+    }
```

Comment:
This only tells us that our pool was created not that there is an active connection to the database... see Slack message.

Reply:
You could grab a connection from the pool, check its status, then release it.  It's a bit more verbose but it would verify that the database connection is happening.

Reply:
oh yeah, i missed that you have to release the connection after you use but yeah I think that ☝️  is the a better test

also we should rename the `grabConnection` method since it does not return a connection is just returns a promise enabled copy of the pool.